velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -1381,3 +1392,84 @@ TEST_F(DateTimeFunctionsTest, parseDatetime) {
       TimestampWithTimezone(86400000, util::getTimeZoneID("Asia/Kolkata")),
       parseDatetime("1970-01-02+00:00", "YYYY-MM-dd+HH:mm"));
 }
+
+TEST_F(DateTimeFunctionsTest, dateFormat) {
```

Comment:
Duplicate test name causes failures.

This somehow causes test failures:
../../velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp:1396:8: error: redefinition of 'class DateTimeFunctionsTest_dateFormat_Test'
 1396 | TEST_F(DateTimeFunctionsTest, dateFormat) {

Reply:
Right, the shortcut version added these tests as well.